client/pages/students.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+
+// TODO(amantri): get this from the server code, preferably auto-generated
+interface Student {
+  student_id: string;
+  first_name: string;
+  last_name: string;
+  assigned_case_manager: string;
+}
```

Comment:
another option is [trpc](https://trpc.io/) which would eliminate needing to duplicate types like this, but not sure how many layers of abstraction the team wants to learn at once

Reply:
Yeah - I'd be inclined to using trpc (or even grpc*) for this. Are the Zapatos generated types an option here?

* we will need to define the json schema for our db somewhere. Protobufs (with json marshallers) are a pretty solid choice here IMHO. jscon-schema looks to be way too verbose.

** aside: there is also grpc-js: https://www.npmjs.com/package/@grpc/grpc-js